railties/lib/rails/generators/erb/scaffold/templates/index.html.erb

```diff
@@ -16,7 +16,7 @@
     <%% @<%= plural_table_name %>.each do |<%= singular_table_name %>| %>
       <tr>
 <% attributes.reject(&:password_digest?).each do |attribute| -%>
-        <td><%%= <%= singular_table_name %>.<%= attribute.name %> %></td>
+        <td><%%= <%= singular_table_name %>.<%= attribute.name %><% if attribute.reference? %>.try(:id)<% end %> %></td>
```

Comment:
I think its safe to do `&.id` instead of `.try(:id)` here. Same with the show template.

Reply:
Thinking a little further on this, we could also try to guess a display name property [similar to ActiveAdmin](https://github.com/activeadmin/activeadmin/blob/26b7c841073cdddbdc8307ae821711654d4ac2c4/lib/active_admin/application_settings.rb#L27) but that might result in some overhead.